<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jointOrientWindow</class>
 <widget class="QDialog" name="jointOrientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joint Orient</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="jointOrientVerticalLayout">
    <item>
     <widget class="QGroupBox" name="aimGroupBox">
      <property name="title">
       <string>Aim Locator</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>279</width>
         <height>44</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="aimGridLayout">
        <item row="1" column="2">
         <widget class="QRadioButton" name="aimNegZRadio">
          <property name="text">
           <string>-z</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="aimNegXRadio">
          <property name="text">
           <string>-x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="aimNegYRadio">
          <property name="text">
           <string>-y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="aimXRadio">
          <property name="text">
           <string>x</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="aimYRadio">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="aimZRadio">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="upGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Up locator</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>279</width>
         <height>39</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="upGridLayout">
        <item row="1" column="2">
         <widget class="QRadioButton" name="upNegZRadio">
          <property name="text">
           <string>-z</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="upNegXRadio">
          <property name="text">
           <string>-x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="upNegYRadio">
          <property name="text">
           <string>-y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="upXRadio">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="upYRadio">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="upZRadio">
          <property name="text">
           <string>z</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="createHorizontalLayout">
      <item>
       <widget class="QCheckBox" name="aimCheckBox">
        <property name="text">
         <string>aim</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="upCheckBox">
        <property name="text">
         <string>up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="axisHorizontalLayout">
      <item>
       <widget class="QPushButton" name="axisOnButton">
        <property name="text">
         <string>Axis On</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="axisOffButton">
        <property name="text">
         <string>Axis Off</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="orientButton">
      <property name="text">
       <string>Orient Joints</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
